example/app/config.yml

```diff
@@ -736,6 +741,46 @@ mhs4:
       logApiGatewayToCloudWatch: true
   logToSharedDestination: '{{SHARED_LOG_DESTINATION_ARN}}'
 
+mhs2:
+  prefix: mhs2
+  prefixNoDash: mhs2
+  buckets:
+    internal:
+      name: mhs2-internal
+      type: internal
+    private:
+      name: mhs2-private
+      type: private
+    protected:
+      name: mhs2-protected
+      type: protected
+    public:
+      name: mhs2-public
+      type: public
+    protected-2:
+      name: mhs2-protected-2
+      type: protected
+    dashboard:
+      name: mhs2-dashboard
+      type: protected
+    shared:
+      name: cumulus-data-shared
+      type: shared
+  system_bucket: mhs2-internal
+  api_distribution_url:
+  ecs:
+    ssh: true
+    keyPairName: mhs-sit
+  apiConfigs:
+    backend:
+      private: true
+      port: 8022
+    distribution:
+      private: true
+      port: 7022
+      logApiGatewayToCloudWatch: true
+  logToSharedDestination: '{{SHARED_LOG_DESTINATION_ARN}}'
+  cma_layer: '{{sit_cma_layer}}'
```

Comment:
So this deployment is for SIT?

Reply:
Yeah mhs2 is just my personal SIT deployment, and I updated it for Lauren and Jon after asking if they wanted me to.